sydent/http/servlets/__init__.py

```diff
@@ -203,22 +182,19 @@ def reqErr(failure: Failure, request: Request) -> None:
             )
         request.finish()
 
+    @functools.wraps(f)
     def inner(*args, **kwargs) -> int:
         """
-        Runs an asynchronous web handler function with the given arguments and add
-        reqDone and reqErr as the resulting Deferred's callbacks.
+        Runs an asynchronous web handler function with the given arguments.
 
+        :param f: The original function passed to the asyncjsonwrap decorator
```

Comment:
`f` isn't a parameter to `inner`, so I think we should leave it out from here!

Reply:
Right it's a param to render. I wasn't paying attention there!